conf/report-template.html

```diff
@@ -346,9 +356,10 @@
       </thead>
         {% for tag, info in database.items() %}
         <tr>
-          <th title="{{info}}"> {{ tag }} </th>
+          <th id="tag"> {{ tag }} </th>
+          <th id="info"> {{ info }} </th>
           {% for tool, tooldata in report.items() %}
-          <th class="{{ tooldata["tags"][tag]["status"] }}
+          <th id="result" class="{{ tooldata["tags"][tag]["status"] }}
```

Comment:
Maybe we could add `text-overflow: ellipsis;` to this? It looks weird if the test boxes do not all have the same height.

Reply:
Btw assuming we do add this, please restore the title (tooltip). We could actually add this to both columns (id + description)

Reply:
BTW. This comment should of course be under the ``{{ info }}`` change.